content/configuration/dev-server.md

```diff
@@ -403,6 +566,11 @@ For example, the dev-server is proxied by nginx, and available on `myapp.test`:
 public: "myapp.test:80"
 ```
 
+Usage via the CLI
+```
+webpack-dev-server --public myapp.test:80
+```
+
 
 ## `devServer.publicPath` 🔑
```

Comment:
Or @SpaceK33z? I'm not sure what they're supposed to indicate. If they were meaningful maybe we can think of a more intuitive display.